lib/pcr.h

```diff
@@ -35,8 +35,35 @@
 
 #include <tss2/tss2_sys.h>
 
+typedef struct tpm2_algorithm tpm2_algorithm;
+struct tpm2_algorithm {
+    int count;
+    TPMI_ALG_HASH alg[8]; //XXX Why 8?
```

Comment:
You should be backporting from the tip of master so you pick up commits like 1bdaef80a8ad7 and 785c7cbc1927

Reply:
I just noticed that [82d987](https://github.com/tpm2-software/tpm2-tools/commit/82d987a8e75ebefec4cbf1c2bd3fda87c2aa5e34) hasn't been backported.  Should I include this commit?

Reply:
Yep

Reply:
Ok, I cherry picked that commit.  The latest should incorporate everything related to this PR from the tip of master.